src/applications/check-in/utils/appointment/index.js

```diff
@@ -220,13 +220,13 @@ const hasPhoneAppointments = appointments => {
  * @returns {Node}
  */
 
-const appointmentIcon = appointment => {
+const appointmentIcon = (appointment, listMode = false) => {
   return (
     <i
       aria-label="Appointment type"
-      className={`fas ${
-        appointment?.kind === 'phone' ? 'fa-phone' : 'fa-building'
-      }`}
+      className={`${
+        listMode && appointment?.kind === 'clinic' ? 'far' : 'fas'
```

Comment:
I'm not sure if this is working for phone. Maybe I'm missing the icon locally? I had to add an appointment to the mocks to test. 
![Screen Shot 2023-02-14 at 11 13 33 AM](https://user-images.githubusercontent.com/2982977/218794615-98a24bbd-f59f-48d2-8d8e-fd11da663c7a.png)
![Screen Shot 2023-02-14 at 11 13 11 AM](https://user-images.githubusercontent.com/2982977/218794624-6540f633-a3b4-4312-9563-4a7827f4ade4.png)
![Screen Shot 2023-02-14 at 11 15 44 AM](https://user-images.githubusercontent.com/2982977/218795014-d3855a18-251f-414e-956c-36c84bfb17fc.png)

Reply:
That is concerning! Let's look at this in our sync today. Also phone is a pre-check-in only thing I added the UUID in my other comment.